scripts/devenv-builder/configure-vm.sh

```diff
@@ -80,8 +80,15 @@ echo -e "${USER}\tALL=(ALL)\tNOPASSWD: ALL" | sudo tee "/etc/sudoers.d/${USER}"
 
 # Check the subscription status and register if necessary
 if ${RHEL_SUBSCRIPTION}; then
+
+    # Handle invalid status
+    if sudo subscription-manager status | grep Invalid >&/dev/null ; then
+        sudo subscription-manager attach --auto
+    fi
+
+    # Assume its unregistered
```

Comment:
This code would not work when the system is unregistered. I'm not sure we should parse the output at all:
```
# subscription-manager status
+-------------------------------------------+
   System Status Details
+-------------------------------------------+
Overall Status: Unknown

System Purpose Status: Unknown

# echo $?
1
```

Reply:
yeah, it wouldnt handle the case when its unregistered, only when its Invalid state (invalid) ,  the other option is to fail early if its Invalid with an error message.

Reply:
So, let's remove `grep` now that we're handing the registration with `auto-attach`?